src/lib/writers/ConfigWriter.ts

```diff
@@ -0,0 +1,29 @@
+
+
+export interface ConfigWriter {
+	write(): void;
+}
+
+export class CompositeConfigWriter implements ConfigWriter {
```

Comment:
I don't think we need a composite writer. We only have yaml format... so there isn't a use case to output to multiple files. So let's not add this in.